TenureInformationApi/V1/Domain/TenuredAsset.cs

```diff
@@ -6,7 +6,7 @@ public class TenuredAsset
     {
         public Guid Id { get; set; }
 
-        public TenuredAssetType Type { get; set; }
+        public TenuredAssetType? Type { get; set; }
```

Comment:
This should really have some test coverage somewhere.

Reply:
I'll merge now to unblock the FE, and will create a ticket for that to be picked by someone later on.